libbpf-tools/trace_helpers.c

```diff
@@ -212,3 +214,21 @@ void print_log2_hist(unsigned int *vals, int vals_size, char *val_type)
 		printf("|\n");
 	}
 }
+
+unsigned long long get_ktime_ns(void)
+{
+	struct timespec ts;
+
+	clock_gettime(CLOCK_MONOTONIC, &ts);
+	return ts.tv_sec * NSEC_PER_SEC + ts.tv_nsec;
```

Comment:
can you add (unsigned long long) cast to ts.tv_sec to avoid overflow on 32-bit architectures?

Reply:
ah, never mind, NSEC_PER_SEC is already unsigned long long